src/lib/configs/project/provider.ts

```diff
@@ -326,8 +330,30 @@ function getDefault(args: Omit<ProviderConfigArgs, "name">) {
       offers: {},
     };
 
-    await addComputePeers(args.noxes, userProvidedConfig);
-    await addOffers(userProvidedConfig);
+    if (userProvidedConfig.env === "local") {
+      userProvidedConfig.computePeers = Object.fromEntries(
+        new Array(args.noxes ?? DEFAULT_NUMBER_OF_LOCAL_NET_NOXES)
+          .fill(null)
+          .map((_, i) => {
+            return [
+              `nox-${i}`,
+              {
+                computeUnits: 1,
+              },
+            ] as const;
+          }),
+      );
```

Comment:
You have lodash, right?
You can use `count(25).` instead of new `Array(25).fill(null).`

Reply:
I always forget I have lodash. I think it's called `times` though, not `count`